README.md

```diff
@@ -1 +1,16 @@
-# life-at-ubc
\ No newline at end of file
+# Life @ UBC
+
+### Installation (if you don't have these already)
+- Node from https://nodejs.org/en/
```

Comment:
Should we include the version of Node we're using in the README as well?

Reply:
I was thinking any recent version would suffice, but I guess to be safe we should all have the same verison. I have 14.9.0 because of 310.